section4/exercises/classMethods.js

```diff
@@ -28,13 +28,24 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
+  }
+  changeProtein(newProtein){
+    this.protein = newProtein;
+  }
+  addTopping(newTopping){
+    this.toppings += ',' + newTopping;
```

Comment:
One note here on the `addTopping` method: The instructions state that `toppings` is an array of strings, so your data type for `this.toppings` will likewise be an array of strings.  This `+=` is a concatenation method for strings only, so you are effectively coercing your array (list) to become a string.  (Javascript can be annoying in this way because it doesn't trigger an error when someone does this).  How could you add the `newTopping` to the array of toppings? Can you think of any array methods to add another item to the list?